govc/session/login.go

```diff
@@ -149,16 +147,30 @@ func (cmd *login) issueToken(ctx context.Context, vc *vim25.Client) (string, err
 	req := sts.TokenRequest{
 		Certificate: c.Certificate(),
 		Userinfo:    cmd.Userinfo(),
-		Delegatable: true,
-		ActAs:       cmd.actas,
+		Renewable:   true,
+		Token:       cmd.token,
 		Lifetime:    cmd.life,
 	}
 
+	if req.Certificate == nil {
+		req.Delegatable = true // Bearer token request
```

Comment:
Hi @dougm,

I like reviewing your PRs because it helps me to appreciate your coding style. I might prefer `req.Delegatable = req.Certificate == nil`, but I respect the way your choice is self-documenting and easy to follow.